README.md

```diff
@@ -34,29 +37,40 @@ Before you begin, you need to install the following tools:
 
 To get started with Scaffold-ETH 2, follow the steps below:
 
-1. Clone this repo & install dependencies
+1. Install from NPM Registry and follow the CLI instructions.
 
 ```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
+npx create-eth@latest
 ```
 
+> 💬 Hint: If you choose Foundry as solidity framework in the CLI, you must have Foundryup previously installed in your machine. Checkout: [Getfoundry.sh](https://getfoundry.sh)
+
 2. Run a local network in the first terminal:
 
 ```
 yarn chain
 ```
 
-This command starts a local Ethereum network using Hardhat. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in `hardhat.config.ts`.
+This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:
+
+- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
+- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
 
 3. On a second terminal, deploy the test contract:
 
 ```
 yarn deploy
 ```
 
-This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `yarn deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
+This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
+
+- Hardhat => `packages/hardhat/contracts`
+- Foundry => `packages/foundry/src`
```

Comment:
This is going to be `packages/foundry/contracts` when we merge this: https://github.com/scaffold-eth/scaffold-eth-2/pull/473